src/components/Organization/OrganizationMenu.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { withRouter, NavLink } from 'react-router-dom';
+import { Menu, Row } from 'antd';
+
+const OrganizationMenu = (props) => {
+  const { children, counts, publishedDataset, hostedDataset, installations, match } = props;
+
+  return (
+    <div style={{ background: 'white' }}>
+      <Row type="flex" justify="start">
+        <Menu
+          style={{ width: 256 }}
+          defaultSelectedKeys={[match.params.section || 'details']}
+          mode="inline"
+        >
+          <Menu.Item key="details">
+            <NavLink to={`/organization/${match.params.key}`}>Overview</NavLink>
```

Comment:
Have you considered using the menu click event instead of NavLinks? That is what we do in the site menu. Also remember that the tabs need to be translatable please.

Reply:
Actually, no. But I'll try so.

Reply:
For tab translations I would suggest to use the same translation key for all tabs across content types. Meaning the translation for identifier/contact etc being the same for organization/dataset etc. If we need to differentiate later, we can split the translations into multiple